src/lib.rs

```diff
@@ -183,6 +217,28 @@ where
     /// Collect range into a container
     ///
     /// Works for any container type that implements [`FromIterator`].
+    /// # Examples
```

Comment:
Can you add an empty line above the headline?

```suggestion
    ///
    /// # Examples
```